Cargo.toml

```diff
@@ -17,5 +17,9 @@ clap = "2.33.0"
 html5ever = "0.24.1"
 lazy_static = "1.4.0"
 regex = "1.3.1"
-reqwest = "0.9.20"
 url = "2.1.0"
+
+[dependencies.reqwest]
+version = "0.10.*"
```

Comment:
My worry is that the wildcard may result in broken builds if a bug slips into one of the patch versions of reqwest. Would it be possible to keep this version number fixed?

Reply:
I understood your concern. I also consider that because other packages' versions are fixed in `Cargo.toml`.

Actually the version of reqwest is fixed to 0.10.0 in `Cargo.lock` here:

https://github.com/rhysd/monolith/blob/reqwest-0.10.0/Cargo.lock#L533

If we don't update `Cargo.lock`, `cargo install` always uses 0.10.0 even if 0.10.1 is released. When running `cargo update`, `Cargo.lock` is modified to use the newer version.

Do you still have some concern?

Reply:
Ah, that's right...
I believe the wildcard there poses no danger if upgrading reqwest is still done manually due to Cargo.lock